app/Console/Kernel.php

```diff
@@ -4,21 +4,20 @@
 
 namespace App\Console;
 
+use App\Services\DataImporterService;
 use Illuminate\Console\Scheduling\Schedule;
 use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
 
 class Kernel extends ConsoleKernel
 {
-    /**
-     * Define the application's command schedule.
-     */
     protected function schedule(Schedule $schedule): void
     {
+        $schedule->call(function (): void {
+            $service = new DataImporterService();
+            $service->run("server");
+        })->everyMinute();
```

Comment:
Import every minute is necessary?

Reply:
It's for testing to see if it works at all.

Reply:
You can use command to test scheduler.
`php artisan schedule:test`

More commands:
`php artisan schedule --help`